src/libostree/ostree-repo.c

```diff
@@ -607,6 +607,34 @@ ostree_repo_class_init (OstreeRepoClass *klass)
 static void
 ostree_repo_init (OstreeRepo *self)
 {
+  /* ABI Size checks for ostree-repo.h, only for LP64 systems;
```

Comment:
This doesn't have to be in `ostree_repo_init` right? Can we just put it at the top of the file?